fe/fe-core/src/test/java/org/apache/doris/datasource/iceberg/IcebergUtilsTest.java

```diff
@@ -60,4 +81,126 @@ private boolean getListAllTables(HiveCatalog hiveCatalog) throws IllegalAccessEx
         declaredField.setAccessible(true);
         return declaredField.getBoolean(hiveCatalog);
     }
+
+    @Test
+    public void testGetMatchingManifest() {
+
+        // partition : 100 - 200
+        GenericManifestFile f1 = getGenericManifestFileForDataTypeWithPartitionSummary(
+                "manifest_f1.avro",
```

Comment:
are these `manifest_f1.avro` real files for test?

Reply:
It is just to construct a name within the manifest object, and there is no real file.